README.md

```diff
@@ -16,4 +16,14 @@ Conventii:
     _old: proiecte create folosind aplicatia AtmelStudio
```

Comment:
De ce nu ai folosit _as sau _atmel?

Reply:
La ce te referi mai exact?

Reply:
In loc de _old

Reply:
Am provizoriu _old pt a indica faptul ca proiectul e facut cu abordarea veche. Dar cred ca ar merge mai bine _atmel.